src/posix/platform/netif.cpp

```diff
@@ -186,7 +187,11 @@ using namespace ot::Posix::Ip6Utils;
 #endif // OPENTHREAD_TUN_DEVICE
 
 #if defined(__linux__)
-static uint32_t sNetlinkSequence = 0; ///< Netlink message sequence.
+#define kDefaultExternalRoutePriority 512
```

Comment:
```suggestion
static const uint32_t kDefaultExternalRoutePriority = 512;
```

Reply:
The reason I defined this as a macro was I saw this block in this file:
```
#define kAddAddress true
#define kRemoveAddress false
#define kUnicastAddress true
#define kMulticastAddress false
```

I thought the advantage of using a macro is, it will be easier to specify a value during compile time. However, I'm inclined to defining it as a constant, if there's no such demand.

Reply:
> it will be easier to specify a value during compile time

Do you mean specifying build configuration?

Reply:
Use a POSIX configuration (ON by default) for this feature? I won't be surprised if someone way to disable this feature.

Reply:
> Do you mean specifying build configuration?
Yes. It seems we don't have a strong demand for including this value in the configuration. Therefore, I made it a constant.